src/reference/asciidoc/endpoint-summary.adoc

```diff
@@ -178,7 +178,7 @@ The following table summarizes the various endpoints with quick links to the app
 | <<./smb.adoc#smb-inbound,SMB Inbound Channel Adapter>>
```

Comment:
I think it won't be bad to mention `SMB Streaming Inbound Channel Adapter` over here as well.
See for example Redis line in this table: it does describe all the channel adapter implementations from that module.

If you do the same fix for (S)FTP modules, it would be great 😄